testing/maas_billing_tests_independent/tests/conftest.py

```
@@ -0,0 +1,251 @@
+    Prefer env override → then cluster policy (POLICY[fallback_key]) → default.
+    Examples:
+      get_limit("RATE_LIMIT_BURST_FREE", "free_burst", 16)
+      get_limit("RATE_LIMIT_BURST_PREMIUM", "premium_burst", 32)
+      get_limit("TOKEN_LIMIT_FREE", "free_tokens", 1000)
+      get_limit("TOKEN_LIMIT_PREMIUM", "premium_tokens", 2000)
+    """
+    v = os.getenv(env_name)
+    if v:
+        try:
+            return int(v)
+        except Exception:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)